### ISSUE TITLE
Typo in latest published WCAG3 draft

### ISSUE THREAD

**Issue by plannero** [w3c/silver#555]
In the currently (June 8th 2021) published draft, I believe there's a missing "not" under section 6.1.1.

It now reads:
"Bronze level means that the content in scope does have any critical errors and meets the minimum percentage"
It should read
"Bronze level means that the content in scope does NOT have any critical errors and meets the minimum percentage"

See https://www.w3.org/TR/2021/WD-wcag-3.0-20210608/#bronze

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
